README.md

```diff
@@ -1 +1,7 @@
-# Hello_ni_bot
\ No newline at end of file
+# Hello_ni_bot
+
+
+		1. The requirements.txt is the list of the packages that need to be install, you can use "pip install -r requirements.txt" to install if you have pip.
```

Comment:
Typo:
be install => be install`ed`

and the last sentence might confuse those who wanna start a slack bot from this repo?
If users don't have `pip`, how can they install those packages?